### DISCUSSION TITLE
Block comments

### DISCUSSION THREAD

**Discussion by adriangalilea** [helix-editor/helix#10136] [Q&A] [answered]
Hello,

I've been trying to use this new feature from `24.03`

I was hoping to be able to comment typescript/react/html code, but when I try to comment:
```
<Foo/>
```

If I do space + c I get the regular comment.
If I do space + C I get  `<F/* o */o/>` around where my cursor was.

But I'm unable to simply comment a line like I do in vscode.

Am I missing something?

**Comment by gabydd** [comment, marked as answer]
The problem here is that the comments don't have enough context to know you are inside jsx, this is covered by https://github.com/helix-editor/helix/issues/7364

**Reply by hiql** [reply]
Any solutions?

**Reply by gabydd** [reply]
You could make the block comment token the jsx one, and then keep the line comment one for js then just use ``<space>c`` when in js or ``<space>C`` in the jsx blocks until that issue is implemented
